internal/bsi/bsi.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
Is it possible to add the installation of the proto generation tool into the Makefile like the other tools (the local `bin/`)? If its not easy (i.e. if its not a go based tool), perhaps we could create a tooling Dockerfile.

Reply:
The invoking of the proto generation should probably be a part of `make generate`

Reply:
Ahh good call - both install and proto generation aren't the most straightforward. Adding that now.